DependencyInjection/FOSCommentExtension.php

```diff
@@ -80,9 +80,11 @@ public function load(array $configs, ContainerBuilder $container)
         // TODO: change it if https://github.com/symfony/DoctrineMongoDBBundle/pull/31 is merged
         if ('mongodb' === $config['db_driver']) {
             if (null === $config['model_manager_name']) {
-                $container->setAlias('fos_comment.document_manager', new Alias('doctrine.odm.mongodb.document_manager', false))->setPublic(true);
+                $container->setAlias('fos_comment.document_manager', new Alias('doctrine.odm.mongodb.document_manager', false));
+                $container->getAlias('fos_comment.document_manager')->setPublic(true);
             } else {
-                $container->setAlias('fos_comment.document_manager', new Alias(sprintf('doctrine.odm.%s_mongodb.document_manager', $config['model_manager_name']), false))->setPublic(true);
+                $container->setAlias('fos_comment.document_manager', new Alias(sprintf('doctrine.odm.%s_mongodb.document_manager', $config['model_manager_name']), false));
+                $container->getAlias('fos_comment.document_manager')->setPublic(true);
```

Comment:
Why are these aliases set public while the constructor was already setting them as private before (and so they were never public even before the 3.4 migration)

Reply:
That's a good question. I only fixed the bc break introduced by the previous commit without thinking about his relevance.
@XWB, do you have an explanation ?